### ISSUE TITLE
Suggestion for gateway / docker: allow reverse proxy address(es).

### ISSUE THREAD

**Issue by ecrode** [MonocleCam/MonocleGateway#5]
The gateway either running directly or via docker requires access to port 443 to work with Alexa.
Issue is that I would like to run it on the same docker instance which is running my nginx proxy.
It's trivial for me to expose the gateway on a custom URL, complete with a valid certificate.
Thus suggestion: allow a gateway to listen on a custom https URL, configurable either via web interface or docker environment variable(s).
In short, I'd like to run the gateway as such:
Alexa > custom URL > nginx > gateway > rtsp...
